src/client/datascience/common.ts

```diff
@@ -150,37 +149,8 @@ export function generateNewNotebookUri(
     }
 }
 
-export async function getRealPath(
-    fs: IFileSystem,
-    execFactory: IPythonExecutionFactory,
-    pythonPath: string,
-    expectedPath: string
-): Promise<string | undefined> {
-    if (await fs.localDirectoryExists(expectedPath)) {
-        return expectedPath;
-    }
-    if (await fs.localFileExists(expectedPath)) {
-        return expectedPath;
-    }
-
-    // If can't find the path, try turning it into a real path.
-    const pythonRunner = await execFactory.create({ pythonPath });
-    const result = await pythonRunner.exec(
-        ['-c', `import os;print(os.path.realpath("${expectedPath.replace(/\\/g, '\\\\')}"))`],
-        {
-            throwOnStdErr: false,
-            encoding: 'utf-8'
-        }
-    );
-    if (result && result.stdout) {
-        const trimmed = result.stdout.trim();
-        if (await fs.localDirectoryExists(trimmed)) {
-            return trimmed;
-        }
-        if (await fs.localFileExists(trimmed)) {
-            return trimmed;
-        }
-    }
+export async function getRealPath(expectedPath: string): Promise<string | undefined> {
```

Comment:
How about we remove this function completely, feels like an unnecessary wrapper around `fs.realpath`. 2 months from now we look at this code we might not understand why we have a wrapper for a simple function.

Reply:
Sure good point.